projects/components/src/utils/test/datagrid/datagrid.wo.ts

```diff
@@ -168,17 +209,24 @@ export class ClrDatagridWidgetObject extends WidgetObject<ClrDatagrid> {
     }
 
     /**
-     * Presses the button at the given {@param index} on the top of the grid.
+     * Gives the text of the button with the given {@param buttonClass}
+     */
+    getButtonText(buttonClass: string): string {
+        return this.getText(`button.${buttonClass}`);
+    }
+
+    /**
+     * Says if the button with the given {@param buttonClass} is disabled.
```

Comment:
Are you sure that adding a {@param} within regular documentation is valid? I haven't heard of it

Reply:
:( this was something that I picked up at my previous job writing backend Java D.B.C. code. But I looked it up, and JSDoc doesn't support it